mathics/core/symbols.py

```diff
@@ -395,25 +412,8 @@ def __new__(cls, name: str, sympy_dummy=None, value=None):
             # value attribute.
             self.sympy_dummy = sympy_dummy
 
-            # This is something that still I do not undestand:
-            # here we are adding another attribute to this class,
-            # which is not clear where is it going to be used, but
-            # which can be different to None just three specific instances:
-            #  * ``System`True``  ->   True
-            #  * ``System`False`` -> False
-            #  * ``System`Null`` -> None
-            #
-            # My guess is that this property should be set for
-            # ``PredefinedSymbol`` but not for general symbols.
-            #
-            # Like it is now, it looks so misterious as
-            # self.sympy_dummy, for which I have to dig into the
-            # code to see even what type of value should be expected
-            # for it.
-            self._value = value
             self._short_name = strip_context(name)
```

Comment:
the short name depends on the evaluation context (i.e., on the Definitions object) so _short_name probably does not belongs here.

Reply:
My understanding is that the short name is the name with all contexts remove. In other words, it is _short_. 

It is like the POSIX `basename` command or `os.path.basename`.   I believe this has always been around. In the back of my mind though, I have been thinking of having print mechanisms that use the short name. 

I don't see this as a big deal one way or another.